aviary/examples/off_design_example.py

```diff
@@ -0,0 +1,224 @@
+    },
+    "cruise": {
+        "subsystem_options": {"core_aerodynamics": {"method": "computed"}},
+        "user_options": {
+            "optimize_mach": True,
+            "optimize_altitude": True,
+            "polynomial_control_order": 1,
+            "use_polynomial_control": True,
+            "num_segments": 1,
+            "order": 3,
+            "solve_for_distance": False,
+            "initial_mach": (0.79, "unitless"),
+            "final_mach": (0.79, "unitless"),
+            "mach_bounds": ((0.79, 0.79), "unitless"),
+            "initial_altitude": (35000.0, "ft"),
+            "final_altitude": (35000.0, "ft"),
+            "altitude_bounds": ((35000.0, 35000.0), "ft"),
+            "throttle_enforcement": "boundary_constraint",
+            "fix_initial": False,
+            "constrain_final": False,
+            "fix_duration": False,
+            "initial_bounds": ((64.0, 192.0), "min"),
+            "duration_bounds": ((60.0, 720.0), "min"),
+        },
+        "initial_guesses": {"time": ([128, 113], "min")},
+    },
+    "descent": {
+        "subsystem_options": {"core_aerodynamics": {"method": "computed"}},
+        "user_options": {
+            "optimize_mach": True,
+            "optimize_altitude": True,
+            "use_polynomial_control": False,
+            "num_segments": 5,
+            "order": 3,
+            "solve_for_distance": False,
+            "initial_mach": (0.79, "unitless"),
+            "final_mach": (0.3, "unitless"),
+            "mach_bounds": ((0.2, 0.8), "unitless"),
+            "initial_altitude": (35000.0, "ft"),
+            "final_altitude": (35.0, "ft"),
+            "altitude_bounds": ((0.0, 35000.0), "ft"),
+            "throttle_enforcement": "path_constraint",
+            "fix_initial": False,
+            "constrain_final": True,
+            "fix_duration": False,
+            "initial_bounds": ((120., 800.), "min"),
+            "duration_bounds": ((5.0, 35.0), "min"),
+            "no_climb": True,
+        },
+        "initial_guesses": {"time": ([241, 30], "min")},
+    },
+    "post_mission": {
+        "include_landing": True,
+        "constrain_range": True,
+        "target_range": (3375.0, "nmi"),
+    },
+}
+
+prob = av.AviaryProblem()
+
+# Load aircraft and options data from user
+# Allow for user overrides here
+prob.load_inputs('models/test_aircraft/aircraft_for_bench_FwFm.csv', phase_info)
+
+# Preprocess inputs
+prob.check_and_preprocess_inputs()
+
+prob.add_pre_mission_systems()
+
+prob.add_phases(phase_info_parameterization=phase_info_parameterization)
+
+prob.add_post_mission_systems()
+
+# Link phases and variables
+prob.link_phases()
+
+prob.add_driver('SNOPT', max_iter=100)
+
+prob.add_design_variables()
+
+# Load optimization problem formulation
+# Detail which variables the optimizer can control
+prob.add_objective()
+
+prob.setup()
+
+prob.set_initial_guesses()
+
+prob.run_aviary_problem()
+prob_fallout = av.AviaryProblem()
```

Comment:
could we get some comments here and/or some sort of horizontal line (ie `#########`) to separate the design, fallout, and alternate sections of the code?